### PR TITLE
Add use_bias kwarg to LSTMCell and GRUCell

### PR DESCRIPTION
**Pull request by lungd**

### CODE REVIEW

**Comment by lungd** [comment]
Waiting for approval

File: src/layers/recurrent.jl
Comment (avik-pal) [inline src/layers/recurrent.jl:226]: We can remove `::Union{ComponentArray, ...}` now from all the calls.

**Review by avik-pal — CHANGES_REQUESTED** [review]
Also let's get a version bump

File: src/layers/recurrent.jl
Comment (lungd) [reply src/layers/recurrent.jl]: Done

**Review by avik-pal — APPROVED** [review]

**Pull request merged**
